spec/mongo/server/push_monitor/connection_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Mongo::Server::PushMonitor::Connection do
+  describe '#check_document' do
+    it 'returns hello document with API version' do
+      meta = Mongo::Server::AppMetadata.new({
+        server_api: { version: '1' }
+      })
+      subject = described_class.new(dup(), {app_metadata: meta})
```

Comment:
What does `dup` do here? You don't need the empty parens for argument-less invocations in Ruby.

Reply:
Good catch, thanks! `double` was meant, corrected.